graylog2-web-interface/src/components/common/SourceCodeEditor.tsx

```diff
@@ -297,11 +332,14 @@ class SourceCodeEditor extends React.Component {
                                $resizable={resizable}>
             <AceEditor ref={(c) => {
               this.reactAce = c;
+              // @ts-ignore
               if (innerRef) { innerRef.current = c; }
             }}
                        annotations={annotations}
+                       // @ts-expect-error
                        editorProps={{ $blockScrolling: 'Infinity' }}
                        // Convert Windows line breaks to Unix. See issue #7889
+                       // @ts-expect-error
                        setOptions={{ newLineMode: 'unix' }}
```

Comment:
I also had look at this error, but it looks like `unix` is the correct value and the types are wrongly expecting a boolean.

Reply:
Yes, there is even an issue for that: https://github.com/securingsincity/react-ace/issues/752